pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -61,7 +61,9 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 			continue
 		}
 		location := vrs.ActiveFileLocation
-		if location == "" {
+		// Volumes in kubevirt eve are of PVC type and managed by kubernetes.
+		// There is no specific filelocation
+		if location == "" && !ctx.hvTypeKube {
```

Comment:
Does this mean:

* if I have no location for my volume file, normally that is an error
* if I am running kubevirt, I don't use "normal" (whatever that means) volumes, so I can handle no location file
?

If so, we need (not in this PR, but sooner rather than later) to abstract it all out. The same way that hypervisors once were a lot of if/then statements, and then became an interface with an implementation for each, we need to do the same for kubevirt.

Reply:
Sure I agree.